nemo/collections/asr/parts/submodules/rnnt_decoding.py

```diff
@@ -319,6 +319,7 @@ def __init__(self, decoding_cfg, decoder, joint, blank_id: int):
                         preserve_frame_confidence=self.preserve_frame_confidence,
                         confidence_method_cfg=self.confidence_method_cfg,
                         loop_labels=self.cfg.greedy.get('loop_labels', False),
+                        use_cuda_graph_decoder=self.cfg.greedy.get('use_cuda_graph_decoder', False),
```

Comment:
I actually want to stick with my original name, "use_cuda_graph_decoder". My concern with "use_cuda_graphs" is that is rather vague. There is nothing stopping a user from wrapping existing nemo code (including this code, since cuda graphs can be children of other cuda graphs!) in a cuda graph, for example.